packages/web3-shared/evm/src/constants/chains.json

```diff
@@ -394,7 +394,8 @@
             "chainId": 42220,
             "name": "CELO",
             "symbol": "CELO",
-            "decimals": 18
+            "decimals": 18,
+            "address": "0x471ece3750da237f93b8e339c536989b8978a438"
```

Comment:
Please also update the token.json.

Reply:
> Please also update the token.json.

It already is.

https://github.com/DimensionDev/Maskbook/blob/c476df99d030a374d180b19111dae2f1a9b4bc9e/packages/web3-constants/evm/token.json#L1395